modules/installing-ocp-agent-ibm-z-kvm.adoc

```diff
@@ -1,18 +1,32 @@
 // Module included in the following assemblies:
 //
 // * installing/installing_with_agent_based_installer/prepare-pxe-infra-agent.adoc
+// * installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc
+
+ifeval::["{context}" == "prepare-pxe-assets-agent"]
+:pxe-boot:
+endif::[]
 
 :_mod-docs-content-type: PROCEDURE
 [id="installing-ocp-agent-ibm-z-kvm_{context}"]
 = Adding {ibm-z-name} agents with {op-system-base} KVM
 
+[NOTE]
+====
+Currently, ISO boot support on {ibm-z-name} (`s390x`) is available only for {op-system-base} KVM, which provides the flexibility to choose either PXE or ISO-based installation.
+For installations with z/VM, only PXE boot is supported.
+
+====
+
```

Comment:
Please move the [NOTE] to after line 21, so the note comes after the abstract but before the Procedure.